### PR TITLE
Revert "Revert "Made event post for techtalk with Monica Beckwith""

### PR DESCRIPTION
**Pull request by annalorimer**
And the award for the worst PR name goes to ... 

tl;dr restoring @fat1996 post for the monica's talk. 

Reverts wics-uw/website#311

### CODE REVIEW

**Comment by fat1996** [comment]
looks good! thanks @annalorimer

**Review by fat1996 — APPROVED** [review]

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Monica Beckwith has worked with the Java Virtual Machine for more than a decade
+not just optimizing the JVM heuristics, but also improving the 
+Just-in-time(JIT) code quality for various processor architectures as well as
+working with the garbage collectors and improving garbage collection for server
+systems.
+
+During this talk, Monica will cover a few JIT and Runtime optimizations and she
+will dive into the HotSpot garbage collection and provide an overview of the
+various garbage collectors available in HotSpot.
+
```
Comment (evykassirer) [inline content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:18]: add eventbrite please - and change it to look more like the other PR you closed + my comments? :3

Comment (annalorimer) [reply content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:18]: My revert should have kept those changes? My bad 😕

Comment (annalorimer) [reply content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:18]: Yeah it's the exact same file that we had originally approved to go up on the site. So ++ adding the eventbrite, but I'm not sure what you mean by making it look like the other PR and your comments.

**Comment by fat1996** [comment]
@annalorimer how do I push to your PR? apparently I'm supposed to have 'push access' if I want to make additions to someone else's PR.

File: content/W2017/W17-event-techtalk-with-Monica-Beckwith.md (hunk 1 above)
Comment (fat1996) [reply content/W2017/W17-event-techtalk-with-Monica-Beckwith.md:18]: I added the abstract, and the out-of-town edits that @evykassirer  suggested.

**Comment by fat1996** [comment]
@annalorimer this can be closed.

**Pull request closed without merging**